### PR TITLE
Adobe Provider

### PR DESCRIPTION
**Pull request by zeo**
This PR merges a brand new provider, Adobe.

### CODE REVIEW

File: src/Adobe/Provider.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    protected function getTokenUrl(): string
+    {
+        return self::BASE_URL.'/token';
+    }
+
+    protected function getTokenFields($code): array
+    {
+        return [
+            ...parent::getTokenFields(),
```
Comment (atymic) [inline src/Adobe/Provider.php:30]: Is this supported in 7.4? Just checking :P

File: src/Adobe/Provider.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace SocialiteProviders\Adobe;
+
+use GuzzleHttp\Exception\GuzzleException;
+use SocialiteProviders\Manager\Contracts\OAuth2\ProviderInterface;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider implements ProviderInterface
```
Comment (atymic) [inline src/Adobe/Provider.php:10]: Can we have an `@see` with a link to the docs please

File: src/Adobe/composer.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "socialiteproviders/adobe",
```
Comment (atymic) [inline src/Adobe/composer.json:2]: Can you please add this to the split script (monorepo builder file in repo root)

**Review by atymic — CHANGES_REQUESTED** [review]

File: src/Adobe/Provider.php (hunk 1 above)
Comment (zeo) [reply src/Adobe/Provider.php:30]: Just checked it out, seems to be supported since PHP 8.0. Will refactor it to support 7.4 syntax!

**Review by atymic — APPROVED** [review]

**Pull request merged**